code/modules/transport/pods/MainWeapon.dm

```diff
@@ -280,8 +284,14 @@
 	current_projectile = new/datum/projectile/bullet/homing/rocket/salvo
 	appearanceString = "pod_weapon_cerberus"
 	firerate = 5 SECONDS
+	shots_to_fire = 3
+	spread = 30
 	icon_state = "cerberus-salvo-rockets"
 
+	post_launch(obj/projectile/P)
+		..()
+		P.proj_data.on_launch(P) // needed as a fix for large pods to set homing target
```

Comment:
Manually calling on_launch() feels a bit cursed. Addnl, any particular reason not to use the standard `alter_projectile` callback pattern for modifying projectiles?